hasura/metadata/remote_schemas.yaml

```diff
@@ -1,5 +1 @@
-- name: vaults
-  definition:
-    url: '{{HASURA_API_BASE_URL}}/remote/vaults'
-    timeout_seconds: 60
-  comment: ""
```

Comment:
same as above

Reply:
https://github.com/coordinape/coordinape/pull/1687/commits/9231d3344b1ebd0eda28693a8f9ab06b9673b416 and https://github.com/coordinape/coordinape/pull/1687/commits/290da405d9db9eec684d05f7c59634c51d5de769